src/components/VideoDetails/VideoDetails.component.jsx

```diff
@@ -1,29 +1,31 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useLocation, useParams } from 'react-router';
 import { Grid } from '@material-ui/core';
 import { getVideos, loadVideo } from '../../services/videoService';
-import SearchContext from '../../providers/SearchContext';
+import { useGlobal } from '../../providers/GlobalContext/GlobalContext';
 import RelatedCard from './RelatedCard';
 
-export default function VideoDetails({ loadVideoFn = loadVideo, getVideosFn = getVideos }) {
+export default function VideoDetails({
+  loadVideoFn = loadVideo,
+  getVideosFn = getVideos,
```

Comment:
Not sure why you would have a these specific defaults, but you should add them to the prop types instead of the function signature. Is there any reason why you want these defaults?, perhaps consider removing the props at all.